### PR TITLE
Update TensorRT-LLM

### PR DESCRIPTION
**Pull request by kaiyux**
* Model Support
  - PyTorch workflow
    - Added Llava-Next support.
    - Added BERT support.
  - Added Qwen2-Audio support. Refer to `examples/qwen2audio/README.md`.
* Features
  - Added Language-Adapter support. Refer to `examples/language_adapter/README.md`.
  - Added AutoDeploy MoE Pattern Matcher for the `fused_moe` op.
  - [BREAKING CHANGE] Added dynamic logits processor support, and deprecated static logits processor in LLM API.
  - Added Deepseek eager mode AllReduce fusion support.
* Bug fixes
  - Fixed incorrect batch slot usage in `addCumLogProbs` kernel. Thanks to the contribution from @aotman in #2787.

### CODE REVIEW

**Review by Shixiaowei02 — APPROVED** [review]

**Comment by vkras** [comment]
With this commit, it's no longer possible to checkout this repo in Windows:
error: cannot stat 'cpp/tensorrt_llm/kernels/trtllmGenKernels/fmha/cubin/FmhaSm100Kernel_QE4m3KvE4m3AccFp32OFp16HQk576HV512HVPerCta512LayoutContiguousKvMaskDenseMultiCtasKvModeVarSeqLenReuseSmemKForVTileSizeQ16TileSizeKv128StaticSwapsMmaAbForGeneration_cubin.cpp': Filename too long

Could you revise those super long file names?
